libs/blocks/src/lib/features/card-feature/index.tsx

```diff
@@ -42,7 +42,7 @@ const Card = ({
     >
       <FluidContainer
         className={qtMerge(
-          'flex flex-col items-center',
+          'flex flex-col items-center lg:px-general-none',
```

Comment:
this fix should be in deriv-com/quill-design#69

Reply:
let's have a chat on this @jia-deriv @prince-deriv , thanks 🙏

Reply:
@jia-deriv can you please check if this is ok now? 
@prince-deriv 🙏

Reply:
Done with removing the class from the `FluidContainer`